packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,39 @@
+Template.videoFlexTab.helpers({
+
+});
+
+
+Template.videoFlexTab.onCreated(function() {
+	this.timeout = null;
+	this.autorun(() => {
+
+	});
+
+	// Opening a PR so we can do this via the https://meet.jit.si/external_api.js
+	$.getScript( "https://cdn.rawgit.com/geekgonecrazy/jitsi-meet/master/external_api.js" )
+		.done(function( script, textStatus ) {
+			var domain = "meet.jit.si"; // Need to get from settings
+			var room = "a124124124124124125125126"; // Need to calc from instance id and room id
+			var width = 500;
+			var height = 500;
+
+			var configOverwrite = {};
+			var interfaceConfigOverwrite = {};
+
+			var api = new JitsiMeetExternalAPI(domain, room, width, height, document.getElementById('videoContainer'), configOverwrite, interfaceConfigOverwrite, true);
+
+			// This for sure needs to be an onReady of some sort instead
+			setTimeout(() => {
+				api.executeCommand('displayName', [Meteor.user().name])
+			}, 3000);
+
+		})
+		.fail(function( jqxhr, settings, exception ) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [There should be no spaces inside this paren.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2700984385/issues/source?bid=3199634&fileBranchId=3342079#l31)